webapp/apps/btax/helpers.py

```diff
@@ -215,101 +211,3 @@ def group_args_to_btax_depr(btax_default_params, asset_yr_str):
             )
     return depr_argument_groups
 
-def btax_results_to_tables(results, first_budget_year):
-    """
-    Take various results from i.e. mY_dec, mX_bin, df_dec, etc
-    Return organized and labeled table results for display
-    """
-    asset_col_meta = dict(btax.parameters.DEFAULT_ASSET_COLS)
-    industry_col_meta = dict(btax.parameters.DEFAULT_INDUSTRY_COLS)
-    r = results[0]
```

Comment:
@talumbau Commenting on the `tax_result[0]` discussion here: See the `r = results[0]` here.  This is how we were doing it (taking the 0th item of the return value from dropq)